Source/MediaInfo/Audio/File_Usac.cpp

```diff
@@ -1743,8 +1848,8 @@ void File_Usac::UsacFrame(size_t BitsNotIncluded)
     {
         switch (C.usacElements[elemIdx].usacElementType&3)
         {
-            case ID_USAC_SCE                                    : UsacSingleChannelElement(); elemIdx=C.usacElements.size()-1; break;
-            case ID_USAC_CPE                                    : UsacChannelPairElement(); elemIdx=C.usacElements.size()-1; break;
+            case ID_USAC_SCE                                    : UsacSingleChannelElement(usacIndependencyFlag); elemIdx=C.usacElements.size()-1; break;

```

Comment:
change to something like if (!coreSbrFrameLengthIndex_Mapping[C.coreSbrFrameLengthIndex].sbrRatioIndex) elemIdx=C.usacElements.size()-1; because parsing of next element should be fine now if there is no SBR.